### PR TITLE
added leetcode solution(19) remove Nth node with code and algortihm

### PR DESCRIPTION
**Pull request by NAVJOT-786**
## Related Issue

[Cite any related issue(s) this pull request addresses. If none, simply state "None”]

Fix: #803 

## Description

This pull request introduces comprehensive solutions to the LeetCode problem "Remove Nth Node From End of List" across multiple programming languages

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [ X] Documentation update
- [ ] Other (specify): _______________

## Checklist
- [X ] I have performed a self-review of my code.
- [ X] I have read and followed the Contribution Guidelines.
- [ X] I have tested the changes thoroughly before submitting this pull request.
- [ X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ X] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

**Comment by NAVJOT-786** [comment]
@Ajay-Dhangar please check the pr

File: dsa-solutions/lc-solutions/0000-0099/0019.Remove-N-Node-From-end-of-list.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        head = head.next;
+        dummy = dummy.next;
+    }
+
+    dummy.next = dummy.next.next;
+
+    return res.next;    
+};
+```
+##Step by Step Algorithm
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0000-0099/0019.Remove-N-Node-From-end-of-list.md:134]: replace `##Step by Step Algorithm` to `## Step by Step Algorithm`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

**Comment by NAVJOT-786** [comment]
@Ajay-Dhangar done with chabges please check

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**